Jenkinsfile

```diff
@@ -0,0 +1,219 @@
+                    echo "You are not authorized to run this job"
+                }
+        
+            }
+        }
+
+        stage("Create an OpenShift project") {
+            steps {
+                script {
+                    openshift.withCluster('operators-test-cluster') {
+                        generateKubeConfig()
+                        openshift.newProject(env.OPENSHIFT_PROJECT_NAME)
+                    }
+                }
+            }
+        }
+
+        stage("Build code binary"){
+            steps{
+                dir("${env.CLONED_REPOSITORY_PATH}") {
+                    script {
+                        sh """
+                        make code/compile
+                        """
+                    }
+                }
+            }
+            post{
+                failure{
+                    echo "====++++'Build code binary' execution failed++++===="
+                    echo "Try to run 'make code/compile' locally and make sure it pass"
+                }
+            }
+        }
+        stage("Build & push container image") {
+            steps{
+                dir("${env.CLONED_REPOSITORY_PATH}") {
+                    script {
+                        withCredentials([usernamePassword(credentialsId: 'quay-aerogear-bot', usernameVariable: 'quayUsername', passwordVariable: "quayPassword")]) {
+                            final String buildConfig = """
+                            {
+                                "apiVersion": "build.openshift.io/v1",
+                                "kind": "BuildConfig",
+                                "metadata": {
+                                    "labels": {
+                                    "build": "${env.OPERATOR_NAME}"
+                                    },
+                                    "name": "${env.OPERATOR_NAME}"
+                                },
+                                "spec": {
+                                    "runPolicy": "Serial",
+                                    "failedBuildsHistoryLimit": 1,
+                                    "successfulBuildsHistoryLimit": 1,
+                                    "source": {
+                                        "binary": {},
+                                        "type": "Binary"
+                                    },
+                                    "strategy": {
+                                        "dockerStrategy": {
+                                            "dockerfilePath": "build/Dockerfile"
+                                        },
+                                        "type": "Docker"
+                                        },
+                                    "output": {
+                                        "to": {
+                                            "kind": "DockerImage",
+                                            "name": "${env.OPERATOR_CONTAINER_IMAGE_NAME}"
```

Comment:
I think this is probably fine, but just note that it's going to overwrite any existing image for that tag on quay.io without having tested it (since that's later in the pipeline here).

Reply:
resolved in [this commit](https://github.com/aerogear/unifiedpush-operator/pull/9/commits/254384960a6683939f464a13355dd044661ec004)

Reply:
LGTM! :+1: